subprojects/veins_testsims/src/veins_testsims/traci/TraCITestApp.cc

```diff
@@ -310,6 +310,18 @@ void TraCITestApp::handlePositionUpdate()
         }
     }
 
+    if (testNumber == testCounter++) {
```

Comment:
this might need updating the corresponding `omnetpp.ini` to execute more tests

Reply:
That's true, I updated the omnetpp.ini accordingly.